lib/Configuration.php

```diff
@@ -131,65 +131,60 @@ public static function loadConfiguration()
             self::reportError('The default configuration file is missing at ' . FILE_CONFIG_DEFAULT);
         }
 
-        Configuration::$config = parse_ini_file(FILE_CONFIG_DEFAULT, true, INI_SCANNER_TYPED);
-        if (!Configuration::$config) {
+        $config = parse_ini_file(FILE_CONFIG_DEFAULT, true, INI_SCANNER_TYPED);
+        if (!$config) {
             self::reportError('Error parsing ' . FILE_CONFIG_DEFAULT);
         }
 
         if (file_exists(FILE_CONFIG)) {
             // Replace default configuration with custom settings
             foreach (parse_ini_file(FILE_CONFIG, true, INI_SCANNER_TYPED) as $header => $section) {
                 foreach ($section as $key => $value) {
-                    Configuration::$config[$header][$key] = $value;
+                    $config[$header][$key] = $value;
                 }
             }
         }
 
-        foreach (getenv() as $envkey => $value) {
+        foreach (getenv() as $envName => $envValue) {
             // Replace all settings with their respective environment variable if available
-            $keyArray = explode('_', $envkey);
+            $keyArray = explode('_', $envName);
             if ($keyArray[0] === 'RSSBRIDGE') {
-                $header = strtolower($keyArray[1]);
-                $key = strtolower($keyArray[2]);
-                if ($value === 'true' || $value === 'false') {
-                    $value = filter_var($value, FILTER_VALIDATE_BOOLEAN);
+                // Preserve casing
+                if ($keyArray[1] == 'TwitterV2Bridge') {
```

Comment:
The actual bug fix is here

Reply:
Huh? I would expect this to be an issue for any bridge, looking at https://github.com/RSS-Bridge/rss-bridge/blob/499d5c2b77def0b9761cf3be68f5981edd4fbe9b/lib/BridgeAbstract.php#L283

I would just try `$bridgeFactory->sanitizeBridgeName($keyArray[1]) ?? $keyArray[1];` to get valid class name here.

Reply:
Yes you are right. There are in total 4 bridges that use config values that could come from env. Also the cache classes yes.